src/awkward/_util.py

```diff
@@ -102,3 +103,14 @@ def unique_list(items: Collection[T]) -> list[T]:
         seen.add(item)
         result.append(item)
     return result
+
+
+def copy_behaviors(existing_class: typing.Any, new_class: typing.Any, behavior: dict):
+    for key, value in behavior.copy().items():
+        if existing_class.__name__ in key:
+            if isinstance(key, str) or "*" in key:
+                continue
```

Comment:
I don't understand; why would singleton-string keys and keys with `"*"` be skipped?

(Also, I prefer if-then to be expressed without jumps, like `continue` or `break`.)